docs/pt_BR/end-to-end_testing.md

```diff
@@ -0,0 +1,50 @@
+# Teste End-to-End
+
+O electron-vue utiliza o [Spectron](http://electron.atom.io/spectron/) o [Mocha](https://mochajs.org/) \(com o [Chai](http://chaijs.com/)\) frameworks de teste do tipo end-to-end. Mocha & Chai APIs, incluem `expect`, `should`, e `assert`, que estão disponíveis no escopo global.
+
+### Rodando testes
+
+```bash
+# Executando o mochajs
+npm run e2e
+```
+
+##### Nota
+
+Antes de ser executado os testes end-to end, o comando `npm run pack` é chamado para criar uma build de produção que o Spectron consome durante os testes.
+
+### Estrutura de arquivos
+
+```
+my-project
+├─ test
+|  ├─ e2e
+│  │  ├─ specs/
+│  │  ├─ index.js
+└─ └─ └─ utils.js
+```
+
+**Na maior parte, você pode ignorar o **`index.js`** e focar somente na escrita **`specs/`**.**
+
+#### `specs/`
+
+Dentro deste diretório é onde os testes são configurados. Graças ao `babel-register`, você possui acesso total ao ES2015.
+
+#### `index.js`
+
+Este arquivo atua como a entrada principal para o Mocha e reunir todos os testes escritos no `specs/` para os testes.
+
+#### `utils.js`
+
+Aqui você poderá encontrar funções genéricas
+Here you will find generic functions that could be of use throughout your `specs/`. Base functions include a `beforeEach` and `afterEach` that handle the electron creation/destruction process.

```

Comment:
- Remove line above `Aqui você poderá encontrar funções genéricas`
- Change line 40 to: Aqui você irá encontrar funções genéricas que podem ser usadas em seu `specs/`. A base inclui funções de `beforeEach` e `afterEach` que manuseiam a criação/destruição do processo do electron.